katcp/ioloop_manager.py

```diff
@@ -52,7 +55,10 @@ def managed(self):
     def get_ioloop(self):
         if not self._ioloop:
             if self._ioloop_managed:
-                self.set_ioloop(tornado.ioloop.IOLoop())
+                if sys.version_info[0] == 3:
+                    self.set_ioloop(AsyncIOLoop())
+                else:
+                    self.set_ioloop(tornado.ioloop.IOLoop())
             else:
                 self.set_ioloop(tornado.ioloop.IOLoop.current())
         return self._ioloop
```

Comment:
Please add a test for this new functionality.
I tried this out in `test_ioloop_manager.py`'s `test_ThreadsafeMethodAttrWrapper` class.  It could be expanded, or we could even make a new test class for `IOLoopManager` that just tries the managed and unmanaged variations.

```python
    def test_managed_io_loop_is_asyncio_in_python3(self):
        if sys.version_info[0] == 3:
            self.assertTrue(hasattr(self.ioloop, "asyncio_loop"))
        else:
            self.assertFalse(hasattr(self.ioloop, "asyncio_loop"))
```